play.pokemonshowdown.com/src/panel-teambuilder.tsx

```diff
@@ -463,11 +487,17 @@ class TeambuilderPanel extends PSRoomPanel<TeambuilderRoom> {
 				<p>
 					<button data-cmd="/newteam" class="button big"><i class="fa fa-plus-circle" aria-hidden></i> New Team</button> {}
 					<button data-cmd="/newteam box" class="button"><i class="fa fa-archive" aria-hidden></i> New Box</button>
+					<input
```

Comment:
`input type="search"` has a few advantages, one of which is that there's a "clear" button, another is that it's cleared if you press Esc.